index.html

```diff
@@ -771,15 +799,33 @@ <h3>
       <h2>
         Security Considerations
       </h2>
-      <div class="issue" title=
-      "Security Considerations section is a work in progress">
+      <div class="issue" title="Security Considerations section is a work in progress">
         <p>
-          This section is a work in progress as this document evolves.
+            This section is a work in progress as this document evolves.
         </p>
+      </div>
         <p>
-          The documents listed below outline initial security considerations
-          for Digital Credentials, both broadly and for presentation on the
-          web. Their contents will be integrated into this document gradually.
+            This section provides a few of the security considerations for the Digital Credentials API.
+            Note that there is a separate section for <a href="#privacy-considerations">Privacy Considerations</a>
+        </p>
+        <h3>Introduction</h3>
+        <p>
+            Digital Credentials APIs are part of and integrated into a broader ecosystem related to digital credentials. Therefore, this section do not specify all security considerations, threats, and mitigations of the ecosystem, but only those related to, directly linked to, or influenced by the Digital Credentials API.
+        <p>
+        <p>
+           Digital Credentials APIs mediate the communication of the presentation from a verifier using a web application to the user's wallet, and the issuance of the credential to the user's wallet when the issuer uses a web application. However, there are other elements that come into play with regard to the security aspects of these interactions, e.g.
```

Comment:
Please make sure this uses the same terminology we define or use in the document (e.g., we don't define a "wallet", but we talk about a [=holder=])